cupy/random/generator.py

```diff
@@ -621,6 +622,8 @@ def seed(self, seed=None):
             except NotImplementedError:
                 seed = numpy.uint64(time.clock() * 1000000)
         else:
+            if isinstance(seed, numpy.ndarray):
+                seed = int(hashlib.md5(seed).hexdigest(), 16) % 2 ** 64
```

Comment:
`[:16]` returns 64bits number.
```suggestion
                seed = int(hashlib.md5(seed).hexdigest()[:16], 16)
```